jest.config.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
```

Comment:
I would like to generate reports when the unit test is executed locally.  How about adding jest options to generate reports:
```
  collectCoverage: true,
  coverageDirectory: "out/unitTestReport",
  coverageReporters: ["json", "html"],
```
here?  When e2e test is executed, html & json reports are crated under `out/e2eTestReport`. I think saving those files in `out/unitTestReport` would be consistent.  What do you think?

Reply:
e2e uses another framework called Mocha. Also, the GHA job expects the coverage to be in `./coverage/` (https://github.com/ansible/vscode-ansible/pull/1028/files#diff-a2115d277b5ca5a2f09a999e53440839cf332b94da177f3d1766334555b0f7c6R26.